src/common/translations/en.json

```diff
@@ -28,7 +28,7 @@
         "addAsset": "Add Asset"
       },
       "history": {
-        "noTransNote": "There is no transaction found in this wallet",
+        "noTransNote": "You have not yet transacted with this asset",
```

Comment:
"You" is subjective compared to starting with "There is".

I'd go with
"There is no transaction found for this asset."

@bitcoinuser also other languages need to be translated based on the update.